brainscore_language/benchmarks/pereira2018_v2022/__init__.py

```diff
@@ -0,0 +1,135 @@
+import logging
```

Comment:
Shouldn't this just replace the current `pereira2018`? I would say we don't want a new file, but rather want to make this the new version.
Tagging @kvfairchild so she can chime in from the plugins perspective.